Makefile

```
@@ -159,6 +159,11 @@ redeploy: load-images # Redploy apex after images changes
 	@kubectl rollout restart deploy/apiserver -n apex
 	@kubectl rollout restart deploy/frontend -n apex
 
+.PHONY: recreate-db
+recreate-db: recreate-db # Delete and bring up a new apex database
```

```suggestion
recreate-db: recreate-db ## Delete and bring up a new apex database
```

It won't show up in help text with a single `#`